src/graphics/aurora/highlightable.h

```diff
@@ -76,6 +78,8 @@ class Highlightable {
 	float _lowerBoundG;
 	float _lowerBoundB;
 	float _lowerBoundA;
+	
```

Comment:
I believe it looks better that way since there are already four groups of members separated by white spaces.
![private_members](https://user-images.githubusercontent.com/7695218/29494320-c67e1cf4-85d1-11e7-9911-76376618c1ad.png)
As for other issues, will fix those right away :)